java/dagger/hilt/processor/internal/HiltCompilerOptions.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package dagger.hilt.android.processor.internal.androidentrypoint;
+package dagger.hilt.processor.internal;
```

Comment:
Moving this file isn't strictly needed but it felt bad to leave it in the entry point package